WalletWasabi.Fluent/Views/Wallets/Send/AdvancedSendOptionsView.axaml

```diff
@@ -0,0 +1,34 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:c="using:WalletWasabi.Fluent.Controls"
+             xmlns:vm="using:WalletWasabi.Fluent.ViewModels.Wallets.Send"
+             mc:Ignorable="d"
+             x:DataType="vm:AdvancedSendOptionsViewModel"
+             x:CompileBindings="True"
+             x:Class="WalletWasabi.Fluent.Views.Wallets.Send.AdvancedSendOptionsView">
+  <c:ContentArea Title="{Binding Title}" Caption=""
+                 EnableBack="{Binding EnableBack}"
+                 EnableNext="True"
+                 NextContent="Done"
+                 EnableCancel="{Binding EnableCancel}"
+                 IsBusy="{Binding IsBusy}"
+                 Height="300" Width="550"
+                 ScrollViewer.VerticalScrollBarVisibility="Auto">
+    <DockPanel>
+      <DockPanel.Styles>
+        <Style Selector="DockPanel > Label">
+          <Setter Property="MinWidth" Value="130" />
+        </Style>
+      </DockPanel.Styles>
+
+      <!-- Custom Fee -->
+      <DockPanel DockPanel.Dock="Top" Margin="0,0,0,10">
+        <Label DockPanel.Dock="Left" Content="_Transaction Fee:" Target="transactionFeeTb" />
```

Comment:
This should say `fee rate` instead of `fee`:

```suggestion
        <Label DockPanel.Dock="Left" Content="_Transaction fee rate:" Target="transactionFeeTb" />
```

Reply:
I am not sure, lets get back here once there is a decision here https://github.com/zkSNACKs/WalletWasabi/pull/6749

Reply:
in this case fee rate is ok, because its in advanced options. (IMO)

Reply:
There is nothing advanced about 'fee rate'. Fee and Fee rate are separate things with different meaning. Correct terms should be used at all places.

Reply:
I would really avoid long titles.
`Fee rate` instead of `Transaction fee rate`?

Or the layout could be changed...
From this
```
Title [              Input field                  ]
```
to this
```
Title 
[              Input field                  ]
```

Reply:
IMO changing the layout is better, given that we have space in that dialog.